.github/workflows/push.yml

```diff
@@ -0,0 +1,74 @@
+name: on push or pull_request
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - version: 8.x
```

Comment:
Node.js 8 is now End-of-Life so I'd have no objections to dropping it from the matrix. Same with Node.js 11 (which should probably be replaced with 12 and 13).

Otherwise LGTM.

Reply:
Yes, I'll do it in a follow up.